src/test/java/org/itsallcode/openfasttrace/matcher/MultilineTextMatcher.java

```diff
@@ -118,4 +120,17 @@ public static MultilineTextMatcher matchesAllLines(final String text)
     {
         return new MultilineTextMatcher(text);
     }
+
+    private List<String> splitPreservingNewLines(final String text)
+    {
+
+        final String lineSplittingRegEx = "(?<=" + LINE_ENDING + ")";
+        final List<String> lines = new ArrayList<>();
+        for (String line : text.split(lineSplittingRegEx))
+        {
+            line = line.replace('\n', '\u240A').replace('\r', '\u240D').replace('\t', '\u2409');
+            lines.add(line);
+        }
```

Comment:
Replace for loop with stream and map()

Reply:
Hammer. Nail. :-)

Also I might move this to a separate small project, because it is universally useful. It should then be Java 8 compatible.